frontend/public/components/edit-yaml.jsx

```diff
@@ -73,8 +73,17 @@ export const EditYAML = connect(stateToProps)(
 
       // Retrieve k8s API spec for autocompletion
       if (!window.sessionStorage.getItem(`${window.SERVER_FLAGS.consoleVersion}--swagger.json`)) {
-        coFetchJSON('api/kubernetes/swagger.json')
-          .then(swagger => window.sessionStorage.setItem(`${window.SERVER_FLAGS.consoleVersion}--swagger.json`, JSON.stringify(swagger)));
+        coFetchJSON('api/kubernetes/swagger.json').then(swagger => {
+          // Only store definitions to reduce the document size.
+          const {definitions} = swagger;
+          const json = JSON.stringify({definitions});
+          try {
+            window.sessionStorage.setItem(`${window.SERVER_FLAGS.consoleVersion}--swagger.json`, json);
```

Comment:
Need to change the [YAML editor autocomplete code](https://github.com/openshift/console/blob/master/frontend/public/module/k8s/autocomplete.ts#L142) to read from the new format.